### DISCUSSION TITLE
Contiki 4.9 Docker does not complete

### DISCUSSION THREAD

**Discussion by rahav** [contiki-ng/contiki-ng#2557] [Q&A] [answered]
Attempting to install Contiki Docker ([Contiki installation guide](https://docs.contiki-ng.org/en/release-v4.9/doc/getting-started/Docker.html)), on Ubuntu 22.04.

The command `git submodule update --init --recursive` hangs on:

     Cloning into '/home/rahav/contiki-ng/arch/cpu/gecko/lib/gecko_sdk'...

**Comment by pjonsson** [comment, marked as answer]
The Gecko SDK submoule is 2 GB, so it will take a while to clone even if you are on a fast connection.

There could also be something git-lfs-related, the Github CI does `git lfs install --local` before doing the submodule update.

**Comment by rahav** [comment]
Thank you, you are amazing as always @pjonsson and it was `git-lfs`. On clean Ubuntu it will halt unless it is installed.

Maybe we should add `sudo apt install git-lfs` to the installation instructions.
